R2API/ItemDrop/DropList.cs

```diff
@@ -318,6 +353,13 @@ public class DropList {
             public static List<PickupIndex> ToPickupIndices(IEnumerable<EquipmentIndex> indices) {
                 return indices.Select(PickupCatalog.FindPickupIndex).ToList();
             }
+
+            public static bool IsValidList(IEnumerable<PickupIndex> dropList) {
```

Comment:
should probably also return false if the droplist is empty

Reply:
Based on the changes that were made to my code by iDeath, the drop list should never be empty. Once all the changes to the drop list were applied it will set itself to a list of just PickupIndex.none if the list finished as empty. I believe the purpose of this was so that weightedSelections wouldn't throw errors for being passed empty lists, but I do not know if this achieved the intended result, removing the dependency to hold on to the original drop lists.